src/index.js

```diff
@@ -0,0 +1,195 @@
+"use strict";
+
+let tempF = 0;
+
+const tempValue = document.getElementById("tempValue")
+const increaseButton = document.getElementById("increaseTempControl")
+const decreaseButton = document.getElementById("decreaseTempControl");
+const landscape = document.getElementById("landscape");
+const currentTempButton = document.getElementById("currentTempButton");
+const cityNameElement = document.getElementById("headerCityName");
+const cityNameInput = document.getElementById("cityNameInput");
+const headerCityName = document.getElementById("headerCityName");
+const cityNameReset = document.getElementById("cityNameReset");
+const skySelect = document.getElementById("skySelect");
+const sky = document.getElementById("sky")
+
+function convertToCelsius(fahrenheit) {
+    return Math.round((fahrenheit - 32) * 5 / 9);
+}
+
+function updateTempColor() {
+    let tempC = convertToCelsius(tempF);
```

Comment:
Since you aren't re-assigning the variable `tempC` in this scope, you should use `const` instead of `let`.
```suggestion
    const tempC = convertToCelsius(tempF);
```